deploy/runtime/ui/Battery Dispatch Options FOM.txt

```diff
@@ -438,25 +438,24 @@ function show_hide_charge_options()
 	}
 	else
 	{
-		show('batt_dispatch_auto_can_charge',option != 3);
-		show('batt_dispatch_auto_can_gridcharge', option !=3 );
-		show('batt_dispatch_auto_can_clipcharge', option != 3);
-		if (technology() == 'Fuel Cell') 
-		{ show('batt_dispatch_auto_can_fuelcellcharge', option !=3); }
-		else
-		{ show('batt_dispatch_auto_can_fuelcellcharge', false); }
+		enable('batt_dispatch_auto_can_charge',option != 3);
+		enable('batt_dispatch_auto_can_gridcharge', option !=3 );
```

Comment:
![image](https://user-images.githubusercontent.com/54449384/141401005-b08f38ce-a3c9-4189-961c-142ceee11a69.png)
Fuel Cell Single Owner might need some enable checks

Reply:
![image](https://user-images.githubusercontent.com/54449384/141401128-fecf529e-f4ca-4a6a-8390-689987929e2e.png)
Page load order is not correct for Fuel Cell Single Owner